### ISSUE TITLE
Japanese

### ISSUE THREAD

**Issue by gosugimoto** [johanmattssonm/birdfont#108]
Hi,
Is there a way to add a Japanese Character set?

**Comment by johanmattssonm** [comment]
Yes. search for CJK (Chinese Japanese and Korean), Hiragana or Katakana. I will add something better in the future.

**Comment by gosugimoto** [comment]
How do I do this on the desktop app for Mac? Is it the same?

**Comment by johanmattssonm** [comment]
Go to the overview tab and press command+f.

**Comment by gosugimoto** [comment]
Oh I see it now, thank you!  I didn't know whether you can just load some
glyphs and have a repository.  So with this, I search, select and draw
right?  cool!!
--
iamgo.net

**Comment by johanmattssonm** [comment]
Yes.
